finality-aleph/src/validator_network/outgoing.rs

```diff
@@ -32,10 +33,13 @@ impl<PK: PublicKey, A: Data, ND: Dialer<A>> Display for OutgoingError<PK, A, ND>
                 "communication with {} failed, protocol error: {}",
                 addr, e
             ),
+            TimedOut => write!(f, "dial timeout",),
         }
     }
 }
 
+const DIAL_TIMEOUT: Duration = Duration::from_secs(43);
```

Comment:
You should probably comment on the choice of the value, especially if it's not obvious.

And yes, I'm also sad that this likely means it will have to be changed. :<